projects/resource-library/src/lib/components/resource-reorder/resource-reorder.component.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, OnInit, Input, Output, EventEmitter, ViewChild, ViewEncapsulation} from '@angular/core';
+import * as _ from 'lodash-es';
+import { ConfigService } from '../../services/config/config.service';
+import { EditorService } from '../../services/editor/editor.service';
+import { EditorTelemetryService } from '../../services/telemetry/telemetry.service';
+import { ToasterService } from '../../services/toaster/toaster.service';
+
+
+@Component({
+  selector: 'lib-resource-reorder',
+  templateUrl: './resource-reorder.component.html',
+  styleUrls: ['./resource-reorder.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class ResourceReorderComponent implements OnInit {
```

Comment:
Is this component used in any of the flows?

Reply:
currently, it's used in the library component. Will remove if not required.